rio_cogeo/cogeo.py

```diff
@@ -89,7 +89,7 @@ def cog_translate(
 
                     mem.build_overviews(overviews, Resampling[overview_resampling])
                     mem.update_tags(
-                        OVR_RESAMPLING_ALG=Resampling[overview_resampling].value
+                        OVR_RESAMPLING_ALG=Resampling[overview_resampling].name.upper()
```

Comment:
@vincentsarago at first I disagreed with upper case here, but since we're putting this in the default GDAL domain, it makes sense 👍

Reply:
@sgillies same feeling here ;-)